src/pybind/mgr/dashboard/frontend/src/app/shared/components/submit-button/submit-button.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, ElementRef, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+import * as _ from 'lodash';
+
+/**
+ * This component will render a submit button with the given label.
+ *
+ * The button will disabled itself and show a loading icon when the user clicks
+ * it, usually initiating a request to the server, and it will stay in that
+ * state until the request is finished.
+ *
+ * To indicate that the request failed, returning the button to the enable
+ * state, you need to insert an error in the form with the 'cdSubmitButton' key.
+ * p.e.: this.rbdForm.setErrors({'cdSubmitButton': 'ERR'});
```

Comment:
Maybe use ``this.rbdForm.setErrors({'cdSubmitButton': true});`` as an example.

Reply:
Done.